src/test/java/teammates/test/cases/ui/pagedata/InstructorFeedbacksPageDataTest.java

```diff
@@ -400,4 +405,30 @@ public void testInitWithoutHighlighting() throws Exception{
         return courseIdsOfUser;
     }
         
+    
+    private Map<String, List<String>> getCourseIdToSectionNameMap(List<InstructorAttributes> instructorsForUser, 
+                                                                  Collection<StudentAttributes> allStudent) {
+        Set<String> courseIds = getSetOfCourseIdsFromInstructorAttributes(instructorsForUser);
+        Map<String, List<String>> result = new HashMap<String, List<String>>();
+        
+        for (StudentAttributes student : allStudent) {
+            if (!courseIds.contains(student.course)) {
+                continue;
+            }
+            
+            if (!result.containsKey(student.course)) {
+                List<String> sectionList = new ArrayList<String>();
+                sectionList.add(student.section);
+                result.put(student.course, sectionList);
+            } else {
+                if (result.get(student.course).contains(student.section)) {
+                    continue;
+                }
+                result.get(student.course).add(student.section);
```

Comment:
Why not just negate the condition here instead of continuing?
Also, `allStudents` could be a better name since it's a collection of students.